### PR TITLE
Rename perftest.json and update renovate config

### PR DESCRIPTION
**Pull request by NHAL22**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```
Comment (benouaer) [inline .github/renovate.json:2]: is this needed?

File: app/server/app-configurations.ts [hunk 2]
```diff
@@ -8,7 +8,11 @@ import helmet from 'helmet';
 import config from 'config';
 import { tyaNunjucks } from './controllers/content';
 import { dateFormat } from './utils/dateUtils';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 import { ContentSecurityPolicyOptions } from 'helmet/dist/types/middlewares/content-security-policy';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```
Comment (benouaer) [inline app/server/app-configurations.ts:15]: should these commented out lines be removed?

File: audit.json [hunk 3]
```diff
@@ -1,1156 +1,2317 @@
...
-  "10011_Cookie Without Secure Flag_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10011_Cookie Without Secure Flag_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10054_Cookie without SameSite Attribute_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore",
-  "10054_Cookie without SameSite Attribute_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10027_Information Disclosure - Suspicious Comments_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10109_Modern Web Application_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=en_GET": "ignore",
-  "10104_User Agent Fuzzer_https://sscs-cor.aat.platform.hmcts.net/ruxitagentjs_ICA27QVfgqrux_10245220704125537.js?lng=cy_GET": "ignore"
+<<<<<<< HEAD
```
Comment (benouaer) [inline audit.json:2]: this file seems like it was corrupted in a merge conflict, does it need to change?

File: package.json [hunk 4]
```diff
@@ -33,9 +33,9 @@
     "test:smoke": "cross-env NODE_PATH=. NODE_ENV=test TS_NODE_PROJECT=app/server/tsconfig.json mocha test/smoke/*.ts --grep @smoke --colors --reporter mochawesome  --reporter-options reportDir=smoke-output/reports/smoke,inlineAssets=true --exit",
     "test:audit": "./bin/run-yarn-audit.sh -f yarn-audit-known-issues",
     "test:crossbrowser": "cross-env SSCS_API_URL=http://sscs-tribunals-api-aat.service.core-compute-aat.internal IDAM_API_URL=https://idam-api.aat.platform.hmcts.net S2S_URL=http://rpe-service-auth-provider-aat.service.core-compute-aat.internal S2S_OAUTH2_URL=https://idam-api.aat.platform.hmcts.net NODE_PATH=. NODE_ENV=test codeceptjs run-multiple ${BROWSER_GROUP:-'--all'} -c test/crossbrowser/saucelabs.conf.ts --steps --reporter mocha-multi --verbose",
-    "test:coverage": "nyc report",
+    "test:coverage": "echo Unit Tests disabled check SSCS-11851",
```
Comment (benouaer) [inline package.json:36]: why is this?

File: security.sh [hunk 5]
```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
```
Comment (benouaer) [inline security.sh:1]: why was this file removed?

File: package-lock.json [hunk 6]
```diff
@@ -1,7 +1,10346 @@
 {
   "name": "sscs-cor-frontend",
   "version": "0.0.1",
+<<<<<<< HEAD
```
Comment (benouaer) [inline package-lock.json:4]: this seems like it was corrupted in a merge conflict
also, I don't think we need both package-lock and yarn.lock files. we're supposed to be using yarn, so we shouldn't really have package-lock

**Comment by benouaer** [comment]
- has yarn.lock file been renamed? it shouldn't be and it shouldn't have any changes as there are no package changes

File: app/server/app-configurations.ts (hunk 2 above)
Comment (NHAL22) [reply app/server/app-configurations.ts:15]: I didn't change these files i think this was from when i rebased off master and nightly dev - i can remove it though
